src/interfaces/strategy.ts

```diff
@@ -178,7 +184,13 @@ export class StrategyInterface<T extends ChainId> extends ServiceInterface<T> {
     return result;
   }
 
-  private async fetchVaultsData(): Promise<VaultData[]> {
-    return fetch("https://d28fcsszptni1s.cloudfront.net/v1/chains/1/vaults/all").then(res => res.json());
+  private async fetchVaultsData({ chainId }: { chainId: ChainId }): Promise<VaultData[]> {
+    const res = await fetch(`https://d28fcsszptni1s.cloudfront.net/v1/chains/${chainId}/vaults/all`);
```

Comment:
```suggestion
    const res = await fetch(`https://d28fcsszptni1s.cloudfront.net/v1/chains/${this.chainId}/vaults/all`);
```
So we don't have to pass it in?

Reply:
Could you change this to `https://api.yearn.finance/..` as well so we're not using the cloudfront url, I'm not sure why we have that

Reply:
Could we just use api.yearn.finance... as well as the endpoint instead of the cloudfront one?

Reply:
Good point, missed that the class had the chainId as a property, done with https://github.com/yearn/yearn-sdk/pull/201/commits/edc61557e24adcf0a2703c16a2c547f514ef4b2e

Refactored the endpoint with https://github.com/yearn/yearn-sdk/pull/201/commits/23e0d0bea0d781d5c7e6ffa6a224ee3182c5cbdd